### DISCUSSION TITLE
hey fam how to activate github copilot for student developer pack

### DISCUSSION THREAD

**Discussion by AkhileshKumar10** [community/community#118826] [GitHub Education]
### Select Topic Area

General

### Body

hey fam how to activate github copilot for student developer pack

**Comment by itzzjb** [comment]
You just need to apply for the student developer pack, if the documents get approved you can use Github copilot.  Go to setting section of your GitHub profile and go to the Github Copilot section to setup once you have the developer pack.

**Comment by Jaaziel-Polanco** [comment]
Apply with the university or institute id or something with your name and the institution name, and wait 3-5 days
